themes-default/slim/src/components/display-show.vue

```diff
@@ -47,7 +48,8 @@ export default {
     computed: {
         ...mapState({
             shows: state => state.shows.shows,
-            indexerConfig: state => state.config.indexers.config.indexers
+            indexerConfig: state => state.config.indexers.config.indexers,
+            statusStrings: state => flip(state.statuses.strings)
```

Comment:
just a suggestion, isn't this simpler?
https://github.com/pymedusa/Medusa/blob/7597778ce0c19be4af039877127342eba7e4c316/themes-default/slim/src/components/add-show-options.vue#L239-L248